### DISCUSSION TITLE
How to check if DEV or PROD in `wxt.config.ts`

### DISCUSSION THREAD

**Discussion by lucadalli** [wxt-dev/wxt#560] [Q&A] [answered]
`import.meta.env.DEV` and `import.meta.env.PROD` are not available in `wxt.config.ts` so how does one conditionally change the manifest based on the build target?

**Comment by aklinker1** [comment, marked as answer]
Look at the example here: https://wxt.dev/guide/manifest.html#per-browser-configuration

The manifest field accepts a function, the first argument being a [`ConfigEnv`](https://wxt.dev/api/wxt/interfaces/ConfigEnv.html) object. The field called `mode` that will either be `"development"`, `"production"`, or whatever else you pass for the `--mode` flag.
